package.json

```diff
@@ -2,11 +2,11 @@
   "name": "root",
   "scripts": {
     "anvil": "anvil --fork-url $ANVIL_FORK_URL --fork-block-number 15578840",
-    "build": "pnpm -r --filter '@wagmi/chains' build && pnpm -r --filter '@wagmi/core' build && pnpm -r --filter './packages/**' --filter './references/packages/**' build",
+    "build": "pnpm --filter '@wagmi/chains' build && pnpm --filter '@wagmi/core' build && pnpm -r --filter './packages/**' --filter './references/**' --filter '!./packages/core' --filter '!./references/**/chains' build",
     "changeset:release": "pnpm build && changeset publish",
     "changeset:version": "changeset version && pnpm install --lockfile-only",
     "cjs:release": "node -r esbuild-register ./scripts/cjs.ts",
-    "dev": "pnpm -r --filter '@wagmi/chains' build && pnpm -r --filter '@wagmi/connectors' --filter './packages/**' dev",
+    "dev": "pnpm --filter '@wagmi/chains' build && pnpm -r --filter './packages/**' --filter './references/**' --filter '!./references/**/chains' dev",
```

Comment:
Cleaned up these scripts. Have some dependencies that pnpm is able to map out correctly with topo sort so handling it here.